ibm/service/cos/resource_ibm_cos_bucket.go

```diff
@@ -980,10 +986,21 @@ func resourceIBMCOSBucketUpdate(d *schema.ResourceData, meta interface{}) error
 					writeSet := writeEvent.(bool)
 					activityTracker.WriteDataEvents = &writeSet
 				}
+				if managementEventSet, ok := d.GetOkExists("activity_tracking.0.management_events"); ok {
+					managementEventValue := managementEventSet.(bool)
+					activityTracker.ManagementEvents = &managementEventValue
+				}
+
+				//crn - Optional field
+				if activityMap["activity_tracker_crn"] != nil {
```

Comment:
With above conflicts message u can simply fetch and add no need to chekc for empty string something

Reply:
similar to above we cannot add a conflict.